packages/venia-concept/src/components/Checkout/cart.js

```diff
@@ -1,35 +1,34 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { bool, func, shape, string } from 'prop-types';
 
-import classify from 'src/classify';
+import { mergeClasses } from 'src/classify';
 import CheckoutButton from './checkoutButton';
 import defaultClasses from './cart.css';
 
-class Cart extends Component {
-    static propTypes = {
-        beginCheckout: func.isRequired,
-        classes: shape({
-            root: string
-        }),
-        ready: bool.isRequired,
-        submitting: bool.isRequired
-    };
+const isDisabled = (busy, valid) => busy || !valid;
 
-    render() {
-        const { beginCheckout, classes, ready, submitting } = this.props;
+const Cart = props => {
+    const { beginCheckout, ready, submitting } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
 
-        return (
-            <div className={classes.root}>
-                <div className={classes.footer}>
-                    <CheckoutButton
-                        ready={ready}
-                        submitting={submitting}
-                        submit={beginCheckout}
-                    />
-                </div>
+    const disabled = isDisabled(submitting, ready);
```

Comment:
Do we use `isDisabled` anywhere else? Want to just inline its logic here?

Reply:
Sure. This used to live in `CheckoutButton` but moving it here allowed me to simplify the button API, which IMO isn't super necessary anyways. I may end up replacing it entirely with one of the shared button/trigger components.